File: foodRecipe/my/myRecipeViewController.m
//
//  myRecipeViewController.m
//  foodRecipe
//
//  Created by Jin-young Ko on 2015. 9. 13..
//  Copyright (c) 2015년 Jin-young Ko. All rights reserved.
//

#import "myRecipeViewController.h"
#import "accountManager.h"
#import "storyboard.h"

@implementation myRecipeViewController

- (void)viewDidLoad {
    
    [super viewDidLoad];
    
    if ([accountManager hasToken]) {
        
    }
    else {
        
    }
}

- (void) openLoginPage {
    
    UINavigationController *nc = (UINavigationController*)[myStoryBoard() instantiateViewControllerWithIdentifier:@"loginNavigationController"];
    
    [self presentViewController:nc animated:YES completion:nil];
}

@end

File: foodRecipe/myRecipe/myRecipeViewController.swift
//
//  myRecipeViewController.swift
//  foodRecipe
//
//  Created by zilla on 2015. 7. 11..
//  Copyright (c) 2015년 MakSSe. All rights reserved.
//

import UIKit

class myRecipeViewController: UIViewController {
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        if(accountManager.hasToken()) {
            
        }
        else {
            self.openLoginPage()
        }
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }
    
    override func viewDidAppear(animated: Bool) {
        super.viewDidAppear(animated)
        
    }
    
    override func viewWillAppear(animated: Bool) {
        
        super.viewWillAppear(animated)
        
    }
    
    func openLoginPage() {
        let vc:UINavigationController = myStoryboard().instantiateViewControllerWithIdentifier("loginNavigationController") as! UINavigationController
        self.presentViewController(vc, animated: true, completion: nil)
    }
}
